imap_processing/ultra/l1a/ultra_l1a.py

```diff
@@ -73,27 +135,43 @@ def xarray_aux(decom_ultra_aux: dict):
                 catdesc=key,  # TODO: short and long descriptions
                 fieldname=key,
             ).output()
+            dims = ["Epoch"]
+        # TOF packetdata has multiple dimensions
+        elif key == "PACKETDATA":
+            attrs = dataclasses.replace(
+                ultra_cdf_attrs.ultra_metadata_attrs,
+                catdesc=key,  # TODO: short and long descriptions
+                fieldname=key,
+                label_axis=key,
+                depend_1="Row",
+                depend_2="Column",
+                units="PIXELS",
+            ).output()
+            dims = ["Epoch", "Row", "Column"]
+        # Use metadata with a single dimension for
+        # all other data products
         else:
             attrs = dataclasses.replace(
                 ultra_cdf_attrs.ultra_metadata_attrs,
                 catdesc=key,  # TODO: short and long descriptions
                 fieldname=key,
                 label_axis=key,
             ).output()
+            dims = ["Epoch"]
 
         dataset[key] = xr.DataArray(
             value,
             name=key,
```

Comment:
Technically, in our CDF documentation, we mentioned that variable names would all be lower case with fields separated by underscores.  I'm totally aware that we haven't been doing that so far, including in my IDEX files, but I was copying what was the standard convention for MMS when I wrote that section.  

So for example, "COIN_TYPE" might now be "ultra_coin-type", or more specifically, "ultra_coin-type_aux_l1a"

Reply:
I'll bring this up with the ultra instrument teams. But put as a TODO for now.